modules/tensor_mechanics/src/userobjects/GeneralizedPlaneStrainUserObject.C

```diff
@@ -38,7 +38,9 @@ GeneralizedPlaneStrainUserObject::GeneralizedPlaneStrainUserObject(
     _Cijkl(getMaterialProperty<RankFourTensor>(_base_name + "elasticity_tensor")),
     _stress(getMaterialProperty<RankTwoTensor>(_base_name + "stress")),
     _out_of_plane_pressure(getFunction("out_of_plane_pressure")),
-    _factor(getParam<Real>("factor"))
+    _factor(getParam<Real>("factor")),
+    _residual(1),
+    _jacobian(1)
```

Comment:
Excellent opportunity to use std::array instead. Anytime you know the size at compile time, use std::array!